src/pages/Vaults/Vault/components/IssueRedeemForm/IssueRedeemForm.tsx

```diff
@@ -19,13 +19,13 @@ import {
   WRAPPED_TOKEN_SYMBOL
 } from '@/config/relay-chains';
 import { ForeignAssetIdLiteral } from '@/types/currency';
+import { TreasuryAction } from '@/types/general.d';
```

Comment:
FYI: I moved this `TreasuryAction` type to a general level as it started being used in other places as well.